src/app/menu/content-menu/content-menu.component.ts

```diff
@@ -24,23 +24,8 @@ export class ContentMenuComponent implements OnInit {
   }
 
   changeTitle() {
-    if (this.router.url === '/login' ) {
-      this.title = 'Login';
-    } else if (this.router.url === '/profile/myprofile' ) {
-      this.title = 'Profile';
-    } else if (this.router.url === '/profile/editprofile' ) {
-      this.title = 'Profile';
-    } else if (this.router.url === '/devices/mydevices' ) {
-      this.title = 'Devices';
-    } else if (this.router.url === '/devices/alldevices' ) {
-      this.title = 'Devices';
-    } else if (this.router.url === '/employees' ) {
-      this.title = 'Employees';
-    } else if (this.router.url === '/identify' ) {
-      this.title = 'Identify';
-    } else if (this.router.url === '/places' ) {
-      this.title = 'Places';
-    }
+      const activeRouter = routes.find(route =>   this.router.url.indexOf(route.router) > -1);
```

Comment:
ok, to może zabezpieczyć się lekko żeby nie brało też wartości z końca to `indexOf(.......) === 1` 
przykład:
url wygląda tak: `localhost:4200/profile?q=devices/all`
i chcemy znaleźć `devices/all` zgodnie z regułą `> -1` znajdzie objekt z profile i będzie błąd :)
więc choć tak trzeba się zabezpieczyć że znaleziony ciąg ma być na pozycji 1 (dlaczego na 1? bo jest znak `/`)